op-challenger/fault/abi_test.go

```diff
@@ -72,3 +72,26 @@ func TestBuildFaultAttackData(t *testing.T) {
 
 	require.Equal(t, data, tx.Data())
 }
+
+// TestBuildFaultStepData ensures that the manual ABI packing is the same as going through the bound contract.
+func TestBuildFaultStepData(t *testing.T) {
+	_, opts, _, contract, err := setupFaultDisputeGame()
+	require.NoError(t, err)
+
+	responder, _ := newTestFaultResponder(t, false)
+
+	data, err := responder.buildStepTxData(StepCallData{
+		StateIndex: 1,
+		ClaimIndex: 2,
```

Comment:
`StepCallData` needs to be changed b/c `ClaimIndex` no longer exists

Reply:
The `ClaimIndex` does exist though in the `step` function call in the FaultDisputeGame contract (see [here](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/contracts/dispute/FaultDisputeGame.sol#L74)).

Just pushed a change to remove the `StateIndex` and keep the `ClaimIndex`. I also left a [comment](https://github.com/ethereum-optimism/optimism/pull/6256/files#r1260030145) in your draft pr addressing the removal of the `ClaimIndex` and leaving the `StateIndex`.